combat-trainer.lic

```diff
@@ -2215,6 +2215,11 @@ class TrainerProcess
     when 'App Bundle'
       retreat
       bput("app my bundle quick", 'You scan', 'Appraise what', 'You cannot appraise', 'You can\'t appraise')
+    when /Summon (?<element>Aether|Air|Earth|Electricity|Fire|Water)/
```

Comment:
Should this regex be case insensitive?

Reply:
I thought about that, too, but noticed the other `when` clauses aren't case-insensitive. For example, a couple lines above looks for exactly `'App Bundle'` with capital A and B.

Should we just make all of the patterns be case-insensitive and do the `case/when` on the lowercased variants?

Reply:
I don't see a reason to keep them case sensitive.

It may have just been an oversight, could you do a quick check through the history to see if there's a good reason it was done that way?

Reply:
I dug through `blame` and `history` of this file and I didn't readily see anything specific about these checks needing to be case-sensitive. I'll make it case-insensitive.